widgets/secretProviders/src/SecretProvidersTable.consts.ts

```diff
@@ -1,9 +1,14 @@
 import type { SecretProvidersWidget } from './widget.types';
 
-export const tableRefreshEvent = 'tokens:refresh';
+export const tableRefreshEvent = 'secretProviders:refresh';
 export const dataSortingKeys: Record<string, keyof SecretProvidersWidget.DataSortingKeys> = {
     name: 'name',
     type: 'type',
     createdAt: 'created_at',
     updatedAt: 'updated_at'
 } as const;
+const translationSecretProvider = Stage.Utils.getT('widgets.secretProviders');
+
+export const translateSecretProviders = (key: string, params?: Record<string, any>) => {
+    return translationSecretProvider(key, params);
+};
```

Comment:
I'm not sure about `translationSecretProvider` name (as a function it would be better to use verb in the name).

Can't we actually do it this way:
```suggestion
export const translateSecretProviders = Stage.Utils.getT('widgets.secretProviders');
```
?

Reply:
as I've already replied to @Vorbert-Kruk comment (https://github.com/cloudify-cosmo/cloudify-stage/pull/2315#discussion_r1047056174) I changed the code https://github.com/cloudify-cosmo/cloudify-stage/pull/2315/commits/3d42996b8017bc2c69ea5f010b6a82d9063221bc